tools/validators/instance_validator/tests/fake_instances/GOOD/multi_instances.yaml

```diff
@@ -26,7 +26,7 @@ FCU-1-GUID:
     VLV-2: CONTROLS
     ZONE-G00: FEEDS
   code: FCU-1
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.